python/python/lancedb/table.py

```diff
@@ -1336,7 +1339,7 @@ def embedding_functions(self) -> dict:
     def search(
         self,
         query: Optional[Union[VEC, str, "PIL.Image.Image", Tuple]] = None,
-        vector_column_name: Optional[str] = None,
+        vector_column_name: Optional[str] = "",
```

Comment:
Any specific reason to change None to "" ?

Reply:
no reason just to avoid passing it again, can change as well

Reply:
hmm. IMO it's weird that we use empty string and not None. But that seems to be a pre-existing issue with the other APIs, so this is fine for now.